apps/element-storybook/src/508-compliance.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/addon-docs';
+import storybook_accessibility from '../../../static/storybook_accessibility.png';
+
+<Meta title="BS4 Migration/508 Compliance" />
```

Comment:
This seems less about bootstrap migration and just a general note to the library. I vote to put in on the main level vs being under the BS4 migration section.

Reply:
good point, moved